cmd/image-builder/main.go

```diff
@@ -151,6 +152,10 @@ func cmdManifestWrapper(pbar progress.ProgressBar, cmd *cobra.Command, args []st
 	if err != nil {
 		return nil, err
 	}
+	seedValue, err := cmd.Flags().GetString("seed")
```

Comment:
Could we just use an integer seed here? I.e. just `GetInt64()`? Having it as a int would have the advantage of:
a) removes some code here as we don't need to hash
b) we could make the seed for manifests available somehoe (print, put into the buildlog etc) even if "--seed" is not provided and users could use that int then as input here

Reply:
Sure, I can make it a direct integer instead. I picked hashing because I felt it gives a nicer interface to the user.